### DISCUSSION TITLE
Project board: enable sorting issues by issue dates

### DISCUSSION THREAD

**Discussion by weitzhandler** [community/community#118765] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Please enable sorting issues in the project kanban by: issue creation date, issue/pr update date (for pr last comitted) etc., both ascending and descending. When there are a lot of issues, descending is what you want sometimes, to see the most recent first.

Currently only possible to sort by name which doesn't help a lot, a chronological order is what's needed.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by flavienbwk** [comment]
It's asked for 5 years (2021) in [another thread](https://github.com/orgs/community/discussions/8518).
